src/create_atoms.cpp

```diff
@@ -183,6 +183,8 @@ void CreateAtoms::command(int narg, char **arg)
                        "Only the first set will be used.");
       mode = MOLECULE;
       onemol = atom->molecules[imol];
+      if (atom->molecular == Atom::TEMPLATE && onemol != atom->avec->onemols[0])
```

Comment:
Please move this error check down further (line 304) to be here:
if (onemol->tag_require` && ...) error->all()
put new error check here
blank line
onemol->check_attributes()